clients/cli/src/main/java/org/apache/gravitino/cli/commands/DeleteMetalake.java

```diff
@@ -57,6 +58,14 @@ public void handle() {
     } catch (NoSuchMetalakeException err) {
       System.err.println(ErrorMessages.UNKNOWN_METALAKE);
       return;
+    } catch (MetalakeInUseException inUseException) {
+      System.err.println(
+          metalake
+              + " in use, please disable it first. for example: use shell\n"
+              + "curl -X PATCH -H \"Accept: application/vnd.gravitino.v1+json\" \\\n"
+              + "-H \"Content-Type: application/json\" -d '{\"inUse\": false}' \\\n"
+              + "http://localhost:8090/api/metalakes/"
+              + metalake);
```

Comment:
Rather than telling the user to use using the rest interface, we should add a command to disable/enable a metalake.

Reply:
> Rather than telling the user to use using the rest interface, we should add a command to disable/enable a metalake.

Thank you, @justinmclean, The Disable and enable commands development can left on me.

Reply:
fix